mod.test.ts

```diff
@@ -635,10 +637,13 @@ Deno.test("basic logging test to ensure no errors", async () => {
   $.logDepth = 0;
 });
 
-Deno.test("setting logging", async () => {
+Deno.test("setting logging", () => {
   const test$ = $.build$();
+  // deno-lint-ignore no-explicit-any
```

Comment:
I really dislike this deno lint rule, but I get why people like it. Could we just add this to be ignored in deno.json?